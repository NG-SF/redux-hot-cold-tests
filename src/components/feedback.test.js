import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import {Feedback} from './feedback';

describe('<Feedback />', () => {
    it('Renders without crashing', () => {
        shallow(<Feedback />);
    });

    it('Should match its snapshot', () => {
    const tree = renderer.create(<Feedback />).toJSON();
    expect(tree).toMatchSnapshot();
    });

    it('Renders the text', () => {
        const guessCount = 0;       
        const wrapper = shallow(<Feedback feedback='Guess again' guessCount={guessCount} />);
        expect(wrapper.text()).toEqual('Guess again ');
        expect(wrapper.contains('Guess again')).toEqual(true);
    });
});