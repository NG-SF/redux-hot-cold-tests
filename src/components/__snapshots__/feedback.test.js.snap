// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Feedback /> Should match its snapshot 1`] = `
<h2
  aria-atomic="true"
  aria-live="assertive"
  id="feedback"
  role="status"
>
   
  <span
    className="visuallyhidden"
  >
    Guess again!
  </span>
</h2>
`;
